AirSkyBoat/AirSkyBoat
# Quest Bug Fixes

**_I affirm:_**


## Please enter a player-facing description

In Search of the Truth:
Fixed a bug that prevented the last part of the quest from playing properly, and skipping the end of the story cutscene,

BRD AF Quest Painful Memory:
Fixed a bug that caused Tros to respawn when completing the Bard AF quest "Painful Memory"

## What does this pull request do? (Please be technical)

Closes #3814 - In Search of the Truth

Closes #3816 - Painful Memory

## Steps to test these changes

In Search of the Truth

Complete the quest as normal. After you report the locations of the witnesses, talking to Tressia will not initiate the end sequence of the quest, instead, it will force the player to collect the key item and watch the final cutscene.

Painful Memory
Start the quest, go to Waters_of_Oblivion: !gotoid 17457346 check the water, Kill the NM, Check the waters again to complete the quest.

## Special Deployment Considerations

N/A
